compute_sdk/globus_compute_sdk/sdk/utils/__init__.py

```diff
@@ -69,3 +71,17 @@ def check_version(task_details: dict | None) -> str | None:
                 f"(worker SDK version: {worker_sdk}; worker OS: {worker_os})"
             )
     return None
+
+
+def get_env_var_with_deprecation(
+    current_name: str, old_name: str, default: str | None = None
+) -> str | None:
+    var_old = os.getenv(old_name)
+    if var_old is not None:
+        warnings.warn(
+            f"{old_name} is deprecated and will be removed in a future relase. "
+            f"Use {current_name} instead."
+        )
+
+    returned_default = var_old if var_old is not None else default
```

Comment:
An alternative approach:
```python
if old_name in os.environ:
    warnings.warn(...)
return os.getenv(current_name, os.getenv(old_name, default))
```
In particular, the `environ` is based on the system-passed environment variables, and they can't be null (or `None`).  Empty string, but not None.  Meanwhile `getenv` does _not_ interpret the empty string as `None`, so I submit the above suggestion as a simpler approach.

Reply:
Good suggestion 👍